interfaces/builtin/posix_mq.go

```diff
@@ -256,35 +276,44 @@ func (iface *posixMQInterface) BeforePrepareSlot(slot *snap.SlotInfo) error {
 	}
 
 	// Only ensure that the given path is valid, don't use it here
-	if _, err := iface.getPath(slot, slot.Name); err != nil {
+	if _, err := iface.getPaths(slot, slot.Name); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func (iface *posixMQInterface) generateSnippet(name string, permissions, paths []string) string {
+	var snippet strings.Builder
+	aaPerms := strings.Join(permissions, " ")
+
+	snippet.WriteString(fmt.Sprintf("  # POSIX Message Queue: %s\n", name))
```

Comment:
this comment should tell if this is for a slot or plug. Also because as written would think than name is the queue name but it's actually the slot/plug name.

Reply:
I added this now in [8cf768e](https://github.com/snapcore/snapd/pull/11928/commits/8cf768e16374bd69f2bc0677b26d4f84d30357b9) in a slightly naive way, let me know if you want me to tweak this (happy to do so).